Code/Framework/AzCore/AzCore/Script/ScriptTimePoint.h

```diff
@@ -17,26 +17,26 @@ namespace AZ
 {
     class ReflectContext;
 
-    AZ_TYPE_INFO_SPECIALIZE(AZStd::chrono::system_clock::time_point, "{5C48FD59-7267-405D-9C06-1EA31379FE82}");
+    AZ_TYPE_INFO_SPECIALIZE(AZStd::chrono::steady_clock::time_point, "{5C48FD59-7267-405D-9C06-1EA31379FE82}");
 
 
-    //! Wrapper that reflects a AZStd::chrono::system_clock::time_point to script.
+    //! Wrapper that reflects a AZStd::chrono::steady_clock::time_point to script.
```

Comment:
Are you sure you want to change ScriptTimePoint to use steady_clock instead of system_clock?

Steady_clock's documentation https://en.cppreference.com/w/cpp/chrono/steady_clock says it's not meant to be used to represent a wall clock time, it could be time since boot. This says to mean it wouldn't be useful for tracking times across machines or boot sessions, as well as potentially not useful for printing/displaying the current time.

I'm ont sure how ScriptTimePoint is actually used, but if there are use cases where time is either cached between rebooting a machine, or meant to show a clock's time, it seems better to use system_clock.

Reply:
The ScriptTimePoint is supplied via the OnTick method. It is meant to count clock ticks since the previous call to OnTick